go/pools/resource_pool_test.go

```diff
@@ -500,73 +505,81 @@ func TestIdleTimeout(t *testing.T) {
 	assert.EqualValues(t, 2, p.IdleClosed())
 }
 
-func TestIdleTimeoutWithSettings(t *testing.T) {
+func TestMaxLifetimeTimeout(t *testing.T) {
+	// maxLifetimeTimeout 0
```

Comment:
looks like you have overridden the existing test.
Kindly add new tests and avoid modifying the existing test.

Reply:
It may have been omitted during rebase. Thank you for pointing it out.